tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -386,10 +412,10 @@ const syncAndWaitForFailure = async () => {
   await (await syncRequired()).waitForDisplayed({ timeout: 20000 });
 };
 
-const closeReloadModal = async (shouldUpdate = false, timeout = 5000) => {
+// Increase timeout to 15secs as it sometimes takes a while for the reload modal to display
+const closeReloadModal = async (shouldUpdate = false, timeout = 15000) => {
```

Comment:
Before, it was working okay without a longer timeout. What scenario is requiring more time? I wonder if that scenario can be redesigned by waiting for loaders or something else.